src/main/java/net/mcreator/ui/modgui/BlockGUI.java

```diff
@@ -136,7 +136,7 @@ public class BlockGUI extends ModElementGUI<Block> {
 	private final JCheckBox canRedstoneConnect = L10N.checkbox("elementgui.common.enable");
 
 	private final JComboBox<String> tintType = new JComboBox<>(
-			new String[] { "No tint", "Grass", "Foliage", "Water", "Sky", "Fog", "Water fog" });
+			new String[] { "No tint", "Grass", "Foliage", "Foliage_Birch", "Foliage_Spruce", "Water", "Sky", "Fog", "Water fog" });

```

Comment:
Use `Foliage birch` and `Foliage spruce`, so they are named the same way other options.

Reply:
I wasn't sure if it was able to take a space I will remove the space thanks

Reply:
That change as been pushed.